data/brands/tourism/hotel.json

```diff
@@ -20,6 +20,18 @@
     }
   },
   "items": [
+    {
+      "displayName": "@HOME",
+      "id": "home-779ccb",
+      "locationSet": {"include": ["001"]},
+      "tags": {
+        "brand": "@HOME",
+        "brand:wikidata": "Q135249100",
+        "name": "@HOME",
```

Comment:
I’m unsure if this becomes “At Home” or just “Home” in normal writing, so I left it as the stylized name. By contrast, I left GLō and Vīb as the plain names instead of turning them into the stylized form.